scene/animation/animation_node_state_machine.h

```diff
@@ -183,6 +186,17 @@ class AnimationNodeStateMachine : public AnimationRootNode {
 	void set_node_position(const StringName &p_name, const Vector2 &p_position);
 	Vector2 get_node_position(const StringName &p_name) const;
 
+	void set_node_visibility(const StringName &p_name, const bool &p_visibility);
+	bool get_node_visibility(const StringName &p_name) const;
+
+	// ng = node group
+
+	void set_ng(const StringName &p_name, const int option_index, const int option);
+	PackedInt32Array get_ng(const StringName &p_name) const;
+
+	void set_ng_names(const PackedStringArray &p_layer_names);
+	PackedStringArray get_ng_names() const;
```

Comment:
These need more clear names

Reply:
I have left some comments as "ng = node group" , in header files but I can just write node groups if it is better? Could you confirm that just comments are not clear enough? If so of course I can amend them.

Reply:
Comments should not be used to clarify naming if naming can be better